lib/utilities/reportHandler.js

```diff
@@ -317,6 +348,7 @@ function reportHandler(clientIP, req, res, log) {
                 itemCounts: getObjectCount,
                 crrStats: crrStatsObj,
                 repStatus: results.getReplicationStates,
+                ingStatus: results.getIngestionStates,
```

Comment:
Do you mind changing the name of this key to ingestStatus? Please :) https://github.com/scality/pensieve-api/pull/10/files#diff-65058f891490afbb80c8704444e0af9bR1095

Reply:
:+1: of course